README.md

```diff
@@ -35,29 +35,33 @@ pip install neural-compressor
 > More installation methods can be found at [Installation Guide](https://github.com/intel/neural-compressor/blob/master/docs/source/installation_guide.md). Please check out our [FAQ](https://github.com/intel/neural-compressor/blob/master/docs/source/faq.md) for more details.
 
 ## Getting Started
-### Quantization with Python API
 
-```shell
-# Install Intel Neural Compressor and TensorFlow
```

Comment:
Please keep deps install steps, so user can start with an empty env.

Reply:
Thanks, added in [16bd09d](https://github.com/intel/neural-compressor/pull/1670/commits/16bd09d922694b02f8ded315b3e89f4cf26a1d37)